documentation/modules/rhv-prerequisites.adoc

```diff
@@ -8,11 +8,18 @@
 
 The following prerequisites apply to {rhv-full} migrations:
 
-* You must use a xref:compatibility-guidelines_{context}[compatible version] of {rhv-full}.
-* You must have the {manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificate.
+* To create a provider, you must have at least the `UserRole` and `ReadOnlyAdmin` roles assigned to you. These are the minimum required permissions, however, any other administrator or superuser permissions will also work.
+
+* To migrate virtual machines:
```

Comment:
Looking good overall. The only thing I'm not sure about is if it's clear enough that both sets of permissions had to be given together. The first one is required for the addition of the provider, but it's also important to leave them after for the migration itself.

Reply:
@bkhizgiy I don't understand. Do you need to have `UserRole` and `ReadOnlyAdmin` roles as well as RHV admin permissions (and  `DiskCreator` and `UserVmManager` permissions)  to migrate VMs?

Reply:
@RichardHoch I think I didn't explain myself well; sorry for the confusion.

The first set of permissions, `UserRole` and `ReadOnlyAdmin`, are required for the addition of the provider but also need to remain for the migration itself. That means if the user adds these permissions, then adds the provider, and afterward removes the permissions, the migration will fail.

The second set of permissions is required only for the migration and needs to be granted only to the VMs the user wishes to migrate. However, it should be given in addition to the first set of permissions.

I just thought it might not be clear enough from the current phrasing.